kratos/includes/kratos_parameters.h

```diff
@@ -383,8 +383,8 @@ class Parameters
     {
         rapidjson::Value tmp(value.c_str(), mpdoc->GetAllocator());
         *mpvalue = tmp;
-//         mpvalue->SetString(rapidjson::StringRef(value.c_str()));
-//        mpvalue->SetString(value.c_str(), value.length());
+	//mpvalue->SetString(rapidjson::StringRef(value.c_str()));
```

Comment:
minor: we could directly take out these commented lines

Reply:
ok